### DISCUSSION TITLE
Latest version deployed to Azure as Container App

### DISCUSSION THREAD

**Discussion by cbutler-captech** [danny-avila/LibreChat#3047] [General]
Back in December, I deployed LibreChat using the LibreChatAzureDeployment terraform scripts, but I don't see any updates to it with the RagApi and VectorDB.  I managed to get it running in a ContainerApp environment with 3 containers (Chat, MeiliSearch and Rag api) connected to a Cosmos MongoDB and an Azure PostgreSQL Database with vector support enabled.  Took me quite a while.  Hoping to Terraform it but I am not the best at Terraform and I am tight on time.  Just wanted to note here that is its possible and I think its a better solution than running a linux VM or even spinning up multiple app services as the old one did.

**Comment by R-Prady** [comment]
can you please share the instructions for Azure pg vectordb? im trying to do it using AlloyDB in GCP. would appreciate your help?
